tyr-runner/src/main/java/org/jboss/tyr/model/json/Property.java

```diff
@@ -20,6 +20,10 @@
     public String name;
     public Object value;
 
+    public Property() {
+
+    }
```

Comment:
why is this needed?

Reply:
cause I am reusing this already existing class for jsonb unmarshalling and it requires noargconstructor :) when I remove it I am getting NPE here https://github.com/khermano/tyr/commit/9fe776e81c5361c482c3abf7bda3e18fc90b6b5e#diff-2495a7c9254abaaeb22b96823ce7bb5aR183